catt/controllers.py

```diff
@@ -135,23 +134,23 @@ class PlaybackError(Exception):
 
 class Cache:
     def __init__(self):
-        self.cache_dir = os.path.join(tempfile.gettempdir(), "catt_cache")
+        self.cache_dir = Path(tempfile.gettempdir(), "catt_cache")
         try:
-            os.mkdir(self.cache_dir)
+            self.cache_dir.mkdir()
         except FileExistsError:
             pass
-        self.cache_filename = os.path.join(self.cache_dir, "chromecast_hosts")
+        self.cache_file = Path(self.cache_dir, "chromecast_hosts")
```

Comment:
`self.cache_dir / "chromecast_hosts"`

Reply:
Not a fan of this syntax, but I like your other suggestions, so I'll change that.

Reply:
Really? Damn, the slash is half the reason I liked pathlib in the first place 😛